sandbox/data_science/notebooks/modeling.ipynb

```diff
@@ -0,0 +1,13313 @@
+{
```

Comment:
I hate having these huge files in the repo. Can you clear all the output from the sells and then recommit?

Reply:
Yes, not my highest priority at the moment, but if we settle on this approach then I'll clean up and consolidate the notebooks so they're more readable